### ISSUE TITLE
New map doesn't feel new.

### ISSUE THREAD

**Issue by ComputerOverlord** [TeamPorcupine/ProjectPorcupine#768]
When I play a new map, I don't want to have an O2 generator and a power generator. I just want a room with an airlock.

**Comment by frankitox16** [comment]
I guess the ship it's just a starting point. In a later stage, we'll travel to other planets where you'll begin from scratch, if i'm correct.

**Comment by koosemose** [comment]
Yeah, I'm really hoping the "ship" thing is just intended as a for testing purposes thing. Trying to make a ship with a game meant for building a stationary base just looks terrible, and trying to imply that it's the ship that brings you to your actual starter base just seems pointless and silly when it's just a stationary thing on the map. That would be much better served by either a ship sprite coming by the starter base which you then pop out of or an intro graphic showing the ship dropping you off, or even just text describing what happened and you just start off inside a basic starter base.

**Comment by Mizipzor** [comment]
Wasn't the save game system meant to provide different starting points? Why don't you create the one you want and add it in a pull request? :)

**Comment by kd7uiy** [comment]
It might be worth having a small "safe" space, especially if we start having consequences for there not being any air initially.
